sycl/doc/extensions/experimental/sycl_ext_oneapi_device_global.asciidoc

```diff
@@ -244,7 +244,14 @@ public:
   device_global() = default;
 #endif // __cpp_consteval
 
-  device_global(const device_global &) = delete;
+  // Available if has_property<device_image_scope> is false
+  constexpr device_global(const device_global &other);
+
+  // Available if has_property<device_image_scope> is false and OtherT is
```

Comment:
Since it a ctor, I think that's a good point. How about something like:

```suggestion
  // Available if PropertyListT::has_property<device_image_scope_key> is false and OtherT is
```